app/src/main/java/com/example/dailytaskmanager/MainActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.example.dailytaskmanager;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import java.util.Calendar;
+
+import android.view.View;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.TimePicker;
+import android.app.DatePickerDialog;
+import android.app.TimePickerDialog;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener{
+
+    ImageButton btnDatePicker, btnTimePicker;
+    EditText dateText, timeText;
+    private int mYear, mMonth, mDay, mHour, mMinute;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        btnDatePicker = (ImageButton) findViewById(R.id.dateButtonId);
+        btnTimePicker = (ImageButton) findViewById(R.id.timeButtonId);
+        dateText = (EditText) findViewById(R.id.dateDisplayId);
+        timeText = (EditText) findViewById(R.id.timeDisplayId);
+
+        Calendar c = Calendar.getInstance();
+        dateText.setText(c.get(Calendar.DAY_OF_MONTH) +"-"+c.get(Calendar.MONTH)+"-"+c.get(Calendar.YEAR));
+        timeText.setText(c.get(Calendar.HOUR_OF_DAY) + ":" + c.get(Calendar.MINUTE));
+
+        btnTimePicker.setOnClickListener(this);
+        btnDatePicker.setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View v)
+    {
+        if(v == btnDatePicker) {
+            final Calendar c = Calendar.getInstance();

```

Comment:
replace with LocalDateTime as mentioned above. Also add a comment on why the current date/time needs to be fetched again

Reply:
That shouldn't be necessary.

Now on implementing LocalDateTime, have added variables LocalDate and LocalTime(holding date and time value)  as class variable and can be accessible throughout the class in onClick method too.